deploy/kubernetes/elastic-agent-kubernetes.yaml

```diff
@@ -0,0 +1,155 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: agent-ingest-management-nodescope
+  namespace: kube-system
+  labels:
+    app: agent-ingest-management-nodescope
+    group: ingest-management
+spec:
+  selector:
+    matchLabels:
+      app: agent-ingest-management-nodescope
+  template:
+    metadata:
+      labels:
+        app: agent-ingest-management-nodescope
+        group: ingest-management
+    spec:
+      serviceAccountName: agent-ingest-management
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: agent-ingest-management-nodescope
+          image: docker.elastic.co/beats/elastic-agent:8.0.0-SNAPSHOT
+          command: ["/bin/sh", "-c"]
+          args: [
+            "yes | ./elastic-agent enroll $KIBANA_HOST $FLEET_ENROLLMENT_TOKEN && ./elastic-agent run"
```

Comment:
uhm I wonder if this should be handled by the agent image itself. What will happen here when we are already enrolled but get a pod recreation?